packages/block-library/src/cover/test/edit.js

```diff
@@ -12,6 +17,36 @@ import {
 	selectBlock,
 } from 'test/integration/helpers/integration-test-editor';
 
+jest.mock(
+	'@wordpress/block-editor/src/components/colors-gradients/use-multiple-origin-colors-and-gradients',
+	() => ( {
+		__esModule: true,
+		default: jest.fn( () => ( {
+			disableCustomColors: true,
+			disableCustomGradients: true,
+			colors: [
+				{
+					name: 'Default',
+					colors: [
+						{
+							name: 'Black',
+							slug: 'black',
+							color: '#000000',
+						},
+						{
+							name: 'Cyan bluish gray',
+							slug: 'cyan-bluish-gray',
+							color: '#abb8c3',
+						},
+					],
+				},
+			],
+			gradients: [],
+			hasColorsOrGradients: true,
+		} ) ),
+	} )
+);
```

Comment:
I'll try and clean this up tomorrow.

Reply:
I've avoided mocking the `useMultipleOriginColorsAndGradients` hook in favour of setting explicit editor settings in https://github.com/WordPress/gutenberg/pull/50115/commits/bbac9072cdd1adb834ce5f50db0e63f6e96b75ab. I'm not 100% confident in that approach either so happy to change it if others have suggestions.

Reply:
The new approach matches the actual editor's behavior.